src/distributed_ls/Euclid/euclid_common.h

```diff
@@ -10,6 +10,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <stdbool.h>
```

Comment:
It seems `stdbool.h` is C99, so it is fine, @rfalgout ?

Reply:
C99 is fine.  Thanks!